lib/tasks/admin.rake

```diff
@@ -52,6 +52,15 @@ namespace :admin do
     end
   end
 
+  desc "Import/Update Jurisdiction custom messages"
+  task import_or_update_jurisdiction_messages: :environment do
+    ActiveRecord::Base.transaction do
+      YAML.load_file("config/sara/jurisdiction_messages.yml").each do |jur_name, jur_values|
+        parse_custom_messages(nil, jur_name, jur_values)
+      end
+    end
+  end
+
```

Comment:
This is a pretty minor comment, but I'm wondering if we can have some logging to the console that prints the results of this rake task.

Do we manage the YAML files in config? I.e., is there any chance the YAML could be malformed and YAML.load_file throws an exception?

Reply:
We manage the files for this privately, outside of this repo, so we can trust what we're parsing is good.

Will add some logging

Reply:
Added logging for jurs that have custom messages, and while doing it also realized the `update` call should really be `update!` so any failures are noticed. https://github.com/SaraAlert/SaraAlert/pull/1159/commits/23c5860dbe788d5b5f260e3919c819a8c5d21580